site/pages/search/results-fr.hbs

```diff
@@ -86,3 +86,8 @@
 		</ul>
 	</div>
 </section>
+
+{{!-- Advanced search link --}}
+<div>
+	<p><a href="https://www.canada.ca/fr/sr/srb/sra.html">Recherche avancée</a></p>
```

Comment:
Remove the empty div and replace this: "https://www.canada.ca/fr/sr/srb/sra.html" by "#URL-of-the-advance-search-page"